hub/apps/get-started/uno-simple-photo-viewer.md

```diff
@@ -0,0 +1,622 @@
+
+```csharp
+using Microsoft.UI.Xaml.Media.Imaging;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using Windows.Storage;
+using Windows.Storage.FileProperties;
+using Windows.Storage.Streams;
+
+namespace SimplePhotosUno;
+
+public class ImageFileInfo : INotifyPropertyChanged
+{
+    public ImageFileInfo(ImageProperties properties,
+        StorageFile imageFile,
+        string name,
+        string type)
+    {
+        ImageProperties = properties;
+        ImageName = name;
+        ImageFileType = type;
+        ImageFile = imageFile;
+        var rating = (int)properties.Rating;
+        var random = new Random();
+        ImageRating = rating == 0 ? random.Next(1, 5) : rating;
+    }
+
+    public StorageFile ImageFile { get; }
+
+    public ImageProperties ImageProperties { get; }
+
+    public async Task<BitmapImage> GetImageSourceAsync()
+    {
+        using IRandomAccessStream fileStream = await ImageFile.OpenReadAsync();
+
+        // Create a bitmap to be the image source.
+        BitmapImage bitmapImage = new();
+        bitmapImage.SetSource(fileStream);
+
+        return bitmapImage;
+    }
+
+    public async Task<BitmapImage> GetImageThumbnailAsync()
+    {
+        StorageItemThumbnail thumbnail =
+            await ImageFile.GetThumbnailAsync(ThumbnailMode.PicturesView);
+        // Create a bitmap to be the image source.
+        var bitmapImage = new BitmapImage();
+        bitmapImage.SetSource(thumbnail);
+        thumbnail.Dispose();
+
+        return bitmapImage;
+    }
+
+    public string ImageName { get; }
+
+    public string ImageFileType { get; }
+
+    public string ImageDimensions => $"{ImageProperties.Width} x {ImageProperties.Height}";
+
+    public string ImageTitle
+    {
+        get => string.IsNullOrEmpty(ImageProperties.Title) ? ImageName : ImageProperties.Title;
+        set
+        {
+            if (ImageProperties.Title != value)
+            {
+                ImageProperties.Title = value;
+                _ = ImageProperties.SavePropertiesAsync();
+                OnPropertyChanged();
+            }
+        }
+    }
+
+    public int ImageRating
+    {
+        get => (int)ImageProperties.Rating;
+        set
+        {
+            if (ImageProperties.Rating != value)
+            {
+                ImageProperties.Rating = (uint)value;
+                _ = ImageProperties.SavePropertiesAsync();
+                OnPropertyChanged();
+            }
+        }
+    }
+
+    public event PropertyChangedEventHandler PropertyChanged;
+
+    protected void OnPropertyChanged([CallerMemberName] string propertyName = null) =>
+        PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+}
+```
+
+This class will serve as a model to represent the image files in the `GridView`. In the next sections, we will make a set of changes to these copied files to make them compatible in a multi-platform context.
```

Comment:
Should the solution build and run at this point?
So far you've given a lot of instructions, it would be good for the user to know whether they've done things correctly by either building and/or running the application (even if it doesn't render correctly)